include/tvm/auto_scheduler/compute_dag.h

```diff
@@ -69,7 +69,7 @@ class AccessAnalyzerNode : public Object {
   /*! \brief Store whether the operation is strictly-inlineable
    * (e.g., injective, broadcast and elementwise without reduction, branch or expenive operations)
    */
-  OperationMap<bool> is_strict_inlineable;
+  OperationMap<bool> is_strictly_inlineable;
```

Comment:
"strict-inlineable" is correct. Both "strict-inlinable" or "strictly inlineable" are correct but "strictly-inlinable" is wrong.